perun-openapi/openapi.yml

```diff
@@ -4387,7 +4405,271 @@ paths:
+
+  /json/attributesManager/getRequiredAttributes/s-g-m:
+    get:
+      tags:
+        - AttributesManager
+      operationId: getRequiredAttributesMemberGroupService
+      summary: Returns member-group attributes required by specified service.
+        If workWithUserAttributes == TRUE, then returns also member and user attributes.
+      parameters:
+        - $ref: '#/components/parameters/serviceId'
+        - $ref: '#/components/parameters/groupId'
+        - $ref: '#/components/parameters/memberId'
+        - $ref: '#/components/parameters/workWithUserAttributesOptional'
+      responses:
+        '200':
+          $ref: '#/components/responses/ListOfAttributesResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /json/attributesManager/getRequiredAttributes/s-f:
+    get:
+      tags:
+        - AttributesManager
+      operationId: getRequiredAttributesFacilityService
+      summary: Returns facility attributes required by specified service.
+      parameters:
+        - $ref: '#/components/parameters/serviceId'
+        - $ref: '#/components/parameters/facilityId'
+      responses:
+        '200':
+          $ref: '#/components/responses/ListOfAttributesResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /json/attributesManager/getRequiredAttributes/s-h:
+    get:
+      tags:
+        - AttributesManager
+      operationId: getRequiredAttributesHostService
+      summary: Returns required host attributes.
+      parameters:
+        - $ref: '#/components/parameters/serviceId'
+        - $ref: '#/components/parameters/hostId'
+      responses:
+        '200':
+          $ref: '#/components/responses/ListOfAttributesResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /json/attributesManager/getRequiredAttributes/s-v:
+    get:
+      tags:
+        - AttributesManager
+      operationId: getRequiredAttributesVoService
+      summary: Returns required vo attributes.
+      parameters:
+        - $ref: '#/components/parameters/serviceId'
+        - $ref: '#/components/parameters/voId'
+      responses:
+        '200':
+          $ref: '#/components/responses/ListOfAttributesResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /json/attributesManager/getRequiredAttributes/ss-f:
+    get:
+      tags:
+        - AttributesManager
+      operationId: getRequiredAttributesFacilityServices
+      summary: Returns facility attributes required by specified list of services.
+      parameters:
+        - $ref: '#/components/parameters/serviceIds'
+        - $ref: '#/components/parameters/facilityId'
+      responses:
+        '200':
+          $ref: '#/components/responses/ListOfAttributesResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /json/attributesManager/getRequiredAttributes/ss-r:
```

Comment:
Didn't find this method with this parameters in JavaDoc even tho implementation allow it. Can you please add JavaDoc to this method?